### PR TITLE
Namespace for cookies - Ready for review - 1.9?  Breaking change.

### PR DESCRIPTION
**Pull request by JohnMcLear**
See discussion at https://github.com/ether/etherpad-lite/issues/664

### CODE REVIEW

**Comment by rhansen** [comment]
Please rebase onto current develop.

File: src/static/js/pad_utils.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ function randomString(len)
 }
 
 function createCookie(name, value, days, path){ /* Used by IE */
```
Comment (rhansen) [inline src/static/js/pad_utils.js:42]: Doesn't `readCookie` need to be updated too?

**Comment by rhansen** [comment]
> Please rebase onto current develop.

I rebased your branch onto develop and force-pushed to your fork.

**Comment by JohnMcLear** [comment]
Closing in favor of: https://github.com/ether/etherpad-lite/pull/4423

**Pull request closed without merging**